docs/eventModelling.md

```diff
@@ -0,0 +1,48 @@
+Someone plugs a USB drive into a workstation while a user session is logged in.|```<Install>``` of type ```<Media>```.  ```<User>``` is the user currently logged into the workstation.|
+Someone unplugs a USB drive into a workstation while a user session is not logged in.|```<Uninstall>``` of type ```<Media>```.  There is no  ```<User>```  element.|
+Antivirus product on a workstation with a logged in user detects malware on a file.|```<AntiMalware>``` with sub-element ```<Quarantine>``` |*This will be modelled as ```<Alert>``` in a future version of schema.*
+An IPS appliance blocks an IP address that may be attempting an SQL injection attack.|```<Alert>``` with sub-element ```<IDS>```.  Attacker IP is recorded in ```<Source>``` and target in ```<Destination>```.|
+A database reports that a processing account logs on and performs a backup.|```<Authenticate>``` and then a second event, ideally ```<Process>``` if the backup is contained within the database and control maintained, otherwise ```<Export>```.| It might be unclear that this is a backup operation, e.g. perhaps a query is performed.  In this case ```<Search>``` may be most appropriate for the second event.
+User sends a document to the print queue for immediate printing.|```<Print>``` with ```<Action>``` set to ```CreateJob```|Q: Why not ```StartPrint```? A: Although user intends this to be printed immediately, it might actually be delayed or never actually be printed, e.g. due to an error.
+Print server or network printer reports that it has started printing a document.|```<Print>``` with ```<Action>``` set to ```StartPrint```.|
+Administrator creates a new user.|```<Create>``` of type ```<User>```|
+Administrator creates a new role for power-users.|```<Create>``` of type ```<Group>```|
+Administrator assigns a user to the power-users role.|```<Authorise>``` of type ```<User>``` with ```<AddGroups>``` element assigned appropriately.|
+A user downloads an executable program from a website (reported by web proxy).|```<View>``` of type ```<Resource>``` with ```<MimeType>``` assigned appropriately.|It is not possible for the web proxy to accurately infer what a client did with the accessed resource.  Of course, analytics are likely to use the MIME type information in conjunction with other events to identify potential issues. 
+User defines a new product line within the stock control system.|```<Create>``` of type ```<Object>``` with ```<Type>``` element assigned to an appropriate string literal, e.g. "Product Line"|
+User opens a file from a network share using MSWord.exe.| ```<View>``` of type ```<File>```|*A future update to the schema will allow ```<EventSource>``` to record details of the opening application.  Currently, this must be recorded within ```<Data>``` elements of ```<EventSource>```*
```

Comment:
Worth clarifying the comment such that the event is coming from the OS rather than the application that initiated the view, thus there are two non-human parties involved, the OS and MSWord.